app/components/general/components/bc-footer/macro.njk

```diff
@@ -14,7 +14,7 @@
       <ul data-test-id="legal-panel" class="bc-lh-r bc-body-r bc-list-none nhsuk-u-padding-0 nhsuk-u-padding-bottom-3 nhsuk-width-container">
         <li class='bc-u-display-inline nhsuk-u-margin-0 nhsuk-u-margin-right-2'>Legal</li>
         <li class='bc-u-display-inline nhsuk-u-margin-0 nhsuk-u-margin-right-2'><a href='https://digital.nhs.uk/about-nhs-digital/privacy-and-cookies'>Privacy and Cookies</a></li>
-        <li class='bc-u-display-inline nhsuk-u-margin-0 nhsuk-u-margin-right-2'><a href='/document/terms-of-use.pdf'>Terms and Conditions</a></li>
+        <li class='bc-u-display-inline nhsuk-u-margin-0 nhsuk-u-margin-right-2'><a href='https://buyingcatalogue.digital.nhs.uk/document/terms-of-use.pdf'>Terms and Conditions</a></li>
```

Comment:
Does the relative URL not work here?

Reply:
This document does not seem to exits anywhere but on https://buyingcatalogue.digital.nhs.uk so I do not think there is any harm in hardcoding it, it just ensures the link is valid if the footer is pulled by a project hosted on a different domain.